interp/constrextern.ml

```diff
@@ -176,7 +176,7 @@ let drop_implicits_in_patt cst nb_expl args =
   let rec impls_fit l = function
     |[],t -> Some (List.rev_append l t)
     |_,[] -> None
-    |h::t,(_loc, CPatAtom None)::tt when is_status_implicit h -> impls_fit l (t,tt)
+    |h::t, CAst.{ v = CPatAtom None }::tt when is_status_implicit h -> impls_fit l (t,tt)
```

Comment:
This will not be accepted by `< 4.04` versions of OCaml, you should write `{ CAst.v =` instead.